src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -113,14 +113,16 @@ void OnClick(object? sender, EventArgs e)
 			{
 				using (var builder = new AlertDialog.Builder(Context))
 				{
+					var title = VirtualView.Title ?? string.Empty;
 					if (VirtualView.TitleColor == null)
 					{
-						builder.SetTitle(VirtualView.Title ?? string.Empty);
+						builder.SetTitle(title);
 					}
 					else
 					{
-						var title = new SpannableString(VirtualView.Title ?? string.Empty);
-						title.SetSpan(new ForegroundColorSpan(VirtualView.TitleColor.ToPlatform()), 0, title.Length(), SpanTypes.ExclusiveExclusive);
```

Comment:
`SpannableString.Length()` is supported on android 29.0+

Reply:
See my other comment, this appears to be a bug in the android binding.

Reply:
OK, I will file an issue and suppress the warnings

Reply:
It is probably similar to https://github.com/xamarin/xamarin-android/issues/6962, suppressing with the link to the issue